src/main/java/com/elixr/poc/constants/ApplicationConstants.java

```diff
@@ -0,0 +1,7 @@
+package com.elixr.poc.constants;
+
+public class ApplicationConstants {
```

Comment:
Declare a private constructor using lombok NoArgsConstructor to avoid instantiating the constant class.

Reply:
Updated.